near-sdk-macros/src/core_impl/info_extractor/attr_sig_info.rs

```diff
@@ -33,13 +53,95 @@ pub struct AttrSigInfo {
     pub original_sig: Signature,
 }
 
+// FIXME: Remove once we switch over to AttrSigInfoV2
```

Comment:
Please, create an issue with the details about the next steps (at least high-level) and link it here